cmd/cvc-operator/controller/cstorvolumeclaim.go

```diff
@@ -350,8 +350,15 @@ func distributeCVRs(
 	} else {
 		usablePoolList = getUsablePoolList(volume.Name, poolList)
 	}
+
 	// randomizePoolList to get the pool list in random order
 	usablePoolList = randomizePoolList(usablePoolList)
+
+	// prioritized pool instances matched to the given
+	// nodeName in case of replica affinity is enabled via cstor volume policy
+	if c.isReplicaAffinityEnabled(claim) {
+		usablePoolList = prioritizedPoolList(claim.Publish.NodeID, usablePoolList)
```

Comment:
This will cause an imbalance in the pool i.e I have a storage class without topology and created an application using that storage class with replica antiaffinity now I think CSI-driver will always pick the first node from that list this will cause imbalance the pool.
cc: @payes

Reply:
this comment need to be fixed

Reply:
sorry.. not this.. but, this needs to get into documentation